pkg/api/v1/conditions/types/types.go

```diff
@@ -170,3 +170,19 @@ func (c *ConditionUpdateEvent) MergeExisting(existing *rctypes.Condition) (*rcty
 		CreatedAt:             existing.CreatedAt,
 	}, nil
 }
+
+// ServerProvisionRequest is the request payload to provision a server.
+// Duplicated from FCP without Spec: https://github.com/equinixmetal/facility-controlplane/blob/main/pkg/api/v1/types.go#L388-L402
+type ServerProvisionRequest struct {
+	ServerID             uuid.UUID `json:"serverID,omitempty"`
+	InstanceID           uuid.UUID `json:"instanceID,omitempty"`
+	Hostname             string    `json:"Hostname,omitempty"`
+	ProviderType         string    `json:"ProviderType,omitempty"`
+	OperatingSystemID    string    `json:"OperatingSystemID,omitempty"`
+	InfrastructureID     []string  `json:"InfrastructureID,omitempty"`
```

Comment:
What's `InfrastructureID` reference?

Reply:
Great check! I think `InfrastructureID ` duplicate with `ServerID`.
We can just keep one. I do not have strong preference. 
What do you think?